server/src/test/java/org/elasticsearch/index/fielddata/FieldDataCacheTests.java

```diff
@@ -78,7 +78,11 @@ private SortedSetOrdinalsIndexFieldData createSortedDV(String fieldName, IndexFi
             fieldName,
             CoreValuesSourceType.KEYWORD,
             new NoneCircuitBreakerService(),
-            AbstractLeafOrdinalsFieldData.DEFAULT_SCRIPT_FUNCTION
+            // TODO(stu): get from KeywordFieldMapper
```

Comment:
Would you please explain this TODO? What exactly is missing from this test?

Reply:
Seemed weird to create a new method handle here.  I was thinking maybe we'd get it from keyword rather than creating a new one.

Reply:
Removed